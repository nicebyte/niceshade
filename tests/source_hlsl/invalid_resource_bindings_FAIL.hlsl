// T: simple_texture vs:VSMain ps:PSMain

#include "inc/triangle.hlsl"

[[vk::binding(0, 0)]] uniform Texture2D tex;
[[vk::binding(0, 0)]] uniform sampler samp;

float4 PSMain(Triangle_PSInput ps_in) : SV_TARGET {
  return tex.Sample(samp, ps_in.texcoord);
}

Triangle_PSInput VSMain(uint vid : SV_VertexID) {
  return Triangle(vid, 1.0);
}
