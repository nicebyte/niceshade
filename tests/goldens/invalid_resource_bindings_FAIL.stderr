Attempt to assign a descriptor of different type to slot 0 in set 0 which is already occupied by samp
